modules/openideal_user/openideal_user.module

```diff
@@ -71,15 +76,15 @@ function openideal_user_entity_type_alter(array &$entity_types) {
  * Implements hook_preprocess_HOOK().
  */
 function openideal_user_preprocess_page(&$variables) {
-  if (\Drupal::currentUser()->isAuthenticated()) {
-    $user_id = \Drupal::currentUser()->id();
-    $user = \Drupal::entityTypeManager()->getStorage('user')->load($user_id);
-    $current_route = \Drupal::service('current_route_match')->getRouteName();
+  if (Drupal::currentUser()->isAuthenticated()) {
```

Comment:
I'm curious, what is the purpose of changing `\Drupal` to `Drupal`?
In Drupal core, there is a standard way to use` \Drupal` although it's possible to use just Drupal in the procedural code: https://www.drupal.org/project/drupal/issues/2053489

So why are we not following it?

Reply:
My bad, it's the "silent code cleanup" in PHPSTORM.